src/iv/ivmain.cpp

```diff
@@ -107,9 +107,53 @@ main(int argc, char* argv[])
     mainWin->raise();
     mainWin->activateWindow();
 
+    ustring uexists("exists");
+    std::vector<std::string> extensionsVector; // Vector to hold all extensions
+    auto all_extensions = OIIO::get_string_attribute("extension_list");
+    for (auto oneformat : OIIO::Strutil::splitsv(all_extensions, ";")) { // Split the extensions by semicolon
+        auto format_exts = OIIO::Strutil::splitsv(oneformat, ":", 2);
+        for (auto ext : OIIO::Strutil::splitsv(format_exts[1], ","))
+            extensionsVector.emplace_back(ext); 
+    }
+
+
```

Comment:
I know this is absurdly nit-picky, but would you mind eliminating the extra blanks here? We often use one blank line to separate conceptual steps (like here, between the gathering of all extensions, and the start of handling the images), but I prefer than there not be more than two blank lines in a row *within a function*. I think that extra blanks inside a function doesn't really add to the readability, but can make the function spread out too much vertically and may not all fit in a screenfull. (Though I do, as it turns out, tend to prefer multiple blank lines to separate standalone functions, to visually separate them.)

Reply:
Completely valid, though, does OIIO have a preferred formatter and/or linter for ensuring pre PR consistency?

Reply:
We use clang-format, though I don't think we are enforcing the blank line issue, because our preference is for at most 1 blank line in a row within a function, exactly 3 blank lines between freestanding (top level) functions/methods, and 1 blank line between method declarations within a class declaration, is not a granularity that clang-format appears to support.

Reply:
Okayy, I'll keep this in mind for other PRs(stuff in docs examples and tests one and possibly other ones if I get to complete thr first one!).